Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -120,6 +120,40 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
           Please make sure to read and understand the documentation page on :ref:`taint effects and unmanaged pods<taint_effects>`.
 
+    .. group-tab:: Kubespray
+
+       `Kubespray <https://github.com/kubernetes-sigs/kubespray>`_ requires Python ≥ 3.10 for recent versions. 
+       For environment setup and dependencies installation, see the `Kubespray Ansible documentation <https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible/ansible.md#installing-ansible>`_.
+
+       **Configure cluster:**
+
+       .. code-block:: bash
+
+           # Copy sample inventory
+           cp -rfp inventory/sample inventory/mycluster
```

Comment:
Being unfamiliar with Kubespray, I don't know where this `inventory` directory is located (well I know, but only because I looked at the GH workflow). Can you add the `cd kubespray` before that, please?